tools/json2bin/evo.py

```diff
@@ -82,36 +82,39 @@ def parse_evolutions(table: Sequence, _size: int, _enum: None) -> bytes:
     .register('evolutions', 44, parse_evolutions, optional=j2b.OptionalBehavior.PAD)
 
 
+NATIONAL_DEX_COUNT = 493
+
+# I'm not sure if this is correct? The form indices in-game are offset at NATIONAL_DEX_COUNT, not SPECIES_BAD_EGG
```

Comment:
They are correct; all of the references to these target entries in the `pl_personal` NARC, which has entries 494 and 495 as `Egg` and `Bad Egg`, respectively. Deoxys forms start at 496.

Reply:
Removed the comments